app/javascript/components/util/DeleteDialog.js

```diff
@@ -78,6 +81,26 @@ class DeleteDialog extends React.Component {
               <div className="character-limit-text">{MAX_REASON_LENGTH - this.state.delete_reason_text.length} characters remaining</div>
             </React.Fragment>
           )}
+          {this.props.type === 'Lab Result' && this.props.promptSymptomOnset && (
```

Comment:
do we need to check the type here or can we just check the symptom onset bool?

Reply:
I guess it's not necessary

Reply:
https://github.com/SaraAlert/SaraAlert/pull/861/commits/0b00f9efd83684aa21526ddee93b433c20c8fb79#diff-431caf4d60fa1325346adae1e9dd1b818a55ea030365a593cdf8ebcaaab58dbdL84